_layouts/faqs.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
```

Comment:
How is this layout is different from the `faq-page`? Is it used somewhere else?

Reply:
this was just to have a more general layout that included the header and footer by default (was also handy for the news feed PR). but there is a lot we can clean up in the layouts I think, which I think @hexylena was planning to do afterwards